### PR TITLE
Plotly

### PR DESCRIPTION
**Pull request by bnmajor**
Create interactive plots for each parameter and time step. New features include:

- X and Y parameter values are displayed for the current position when the plot is hovered over
- Plots can be zoomed in and out
- Dashboard defaults to playing
- Pausing will make the `Plotly` toolbar available, which has additional features for view values and manipulating the plot, as well as the ability to save the plot as a `.png`
- Hovering over a selected parameter in the navigation bar will display the minimum and maximum values for the current time step
- The navigation bar is now re-sizable

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@girder/components": "https://github.com/girder/girder_web_components#09eb0bf22bfdbc99f7a4d8e55fb2d33bd1d5fd1a"
+    "@girder/components": "https://github.com/girder/girder_web_components#09eb0bf22bfdbc99f7a4d8e55fb2d33bd1d5fd1a",
```
Comment (cjh1) [inline client/package.json]: Do you think we can just use this package from npm now?

**Comment by cjh1** [comment]
I don't see the tooltip for min and max on a parameter, is that on another branch?

**Comment by bnmajor** [comment]
@cjh1 I had made additional changes on my branch without committing them. That should be corrected now.

**Comment by cjh1** [comment]
@bnmajor I still not seeing the tool tip, let chats I am sure I am missing something.

**Pull request merged**